docs/index.md

```diff
@@ -1,25 +1,49 @@
 
-# Disclaimer
+## Disclaimer
 
 **This project is a work in progress.** Nothing presented in this repo—whether in the source code, issue tracker, or wiki—is a final product unless it is marked as such or appears on [www.consumerfinance.gov](https://www.consumerfinance.gov). In-progress updates may appear on [beta.consumerfinance.gov](https://beta.consumerfinance.gov).
 
-# Technology stack
+## Technology stack
 
 The standard technology stack for development of cfgov-refresh within the CFPB consists of the following base:
 
 - macOS
 - [Homebrew](https://brew.sh) - package manager for installing system software on OSX
-- Python and PIP (Python package manager)
-- [Jinja templates](http://jinja.pocoo.org) for front-end rendering
-- [Wagtail CMS](https://wagtail.io) for content administration
-- Dependencies, listed below
+- [Python 2.7](https://docs.python.org/2.7/) and [pip (Python package manager)](https://pip.pypa.io/en/stable/user_guide/)
+- [Jinja2 templates](http://jinja.pocoo.org/docs/2.10/) for front-end rendering. See [`requirements/libraries.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/libraries.txt) for version.
+- [Wagtail CMS](https://wagtail.io) for content administration. See [`requirements/wagtail.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/wagtail.txt) for version.
+- [PostgreSQL](https://www.postgresql.org/) and [Psycopg](http://initd.org/psycopg/) for our database. See [`requirements/postgres.txt`](https://github.com/cfpb/cfgov-refresh/tree/master/requirements/postgres.txt) for version.
+- Additional dependencies, listed below
 
-# Dependencies
+## Additional dependencies
 
 - [Elasticsearch](https://www.elastic.co):
   Used for full-text search capabilities and content indexing.
-- [Node](http://nodejs.org) and [yarn](https://yarnpkg.com/):
-  Used for downloading and managing front-end dependencies and assets.
+- [Node 8](http://nodejs.org) and [yarn](https://yarnpkg.com/):
+  Used for downloading and managing front-end dependencies and assets. Front-end dependencies are listed in the project's [package.json](https://github.com/cfpb/cfgov-refresh/blob/master/package.json) file.
+- [Gulp 4](https://gulpjs.com/) for running tasks, including compiling front-end assets and running acceptance and unit tests.
+- [virtualenv](https://virtualenv.pypa.io/en/stable/)
+- [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/)
+
+## Versions
+
+Versions for most front-end packages are kept updated in the project's [package.json](https://github.com/cfpb/cfgov-refresh/blob/master/package.json) file.
+
+Versions for back-end software including Python, Jinja, Wagtail, PostgreSQL, etc. are kept in the project's requirements files:
```

Comment:
Technically Python and PostgreSQL versions aren't defined in the requirements files. Maybe change this to "Django, Wagtail, Jinja, etc."?